test/acs-engine-test/config/config.go

```diff
@@ -14,6 +14,7 @@ type Deployment struct {
 	SkipValidation    bool   `json:"skip_validation,omitempty"`
 }
 
+// TestConfig represents a cluster config
```

Comment:
@dmitsh I did my best document the go funcs under test/. Can you double check? Will call out some that I wasn't able to infer about.